velociraptor/autoplotter/lines.py

```diff
@@ -39,9 +39,16 @@ class VelociraptorLine(object):
     bins: unyt_array
     # Scatter can be: "none", "errorbar", or "shaded"
     scatter: str
-    # Output: centers, values, scatter - initialised here to prevent crashes
+    # Boolean. If true, for mean and median lines makes the function create_line
+    # return x and y data points that lie in the bins where the number of data points
+    # is smaller than the minimum value required to compute the mean (or median)
+    # Default: True.
+    return_additional: bool = True
```

Comment:
I don't think we need this as a configurable parameter, it causes too much code confusion. Just pass `return_additional=True` directly to the functions that you know expect it.